casa/IO/MultiFileBase.h

```diff
@@ -53,50 +52,54 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
   // Hence the memory must be freed using free, which makes it impossible
   // to use a shared_ptr to that memory. Hence it is encapsulated in this class.
   // </synopsis>
-  struct MultiFileBuffer {
+  class MultiFileBuffer {
+  public:
     MultiFileBuffer (size_t bufSize, Bool useODirect);
     ~MultiFileBuffer()
-      { if (data) free (data); }
-    // Data member
-    char* data;
+      { if (itsData) free(itsData); }
```

Comment:
FYI in C++17 it's possible to use aligned allocation in C++.

Reply:
I'll leave it for now.